src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -357,6 +357,59 @@ public static class Extensions
             return CS8VerifyAnalyzerAsync(originalCode);
         }
 
+        [Fact]
```

Comment:
Nit: Add `WorkItem` attribute for all added tests?

Reply:
Done